keyboards/plain60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _MA 0
+
+#define _______ KC_TRNS
```

Comment:
This is included when you `#include QMK_KEYBOARD_H`

```suggestion

```

Reply:
Didn't know that - I'll remove it.